guides/common/modules/ref_performing-configuration-management.adoc

```diff
@@ -0,0 +1,16 @@
+[id="performing-configuration-management_{context}"]
+= Performing Configuration Management
+
+Performing configuration management using Puppet in {Project} involves the following high-level steps:
+
+. Managing Puppet modules on the Puppet server, i.e. installing and updating them.
+. Importing Puppet classes and environments from Puppet modules into {Project}.
+. Optional: Creating config groups.
+. Configuring overrides of Smart Class parameters on various levels.
+. Assigning Puppet classes, modules or config groups to host groups or individual hosts.
+. Configuring intervals for runs of the Puppet agent on hosts and for configuration enforcement runs of the Puppet server.
+. Monitoring configuration management via reports in the {ProjectWebUI}.
+. Optional but recommended: Configuring email notifications.
+See {AdministeringDocURL}Configuring_Email_Notifications_admin[Configuring Email Notifications] in _Administering {ProjectName}_.
+
+{Project} then runs configuration management automatically in the configured intervals to enforce Puppet configuration on the managed hosts, or you can initiate it manually on demand with the *Run Puppet Once* feature.
```

Comment:
The automatical run happens only once user deploy the Puppet agent on the host, that typically happens as part of the provisioning process. Otherwise, users must install the agent manually (or automate in the registration template, which should be rather trivial)

Reply:
That's the assumption of this module. I've tried to add a mention into the first paragraph.